src/styles/container.scss

```
@@ -0,0 +1,25 @@
+.container {
+  display: grid;
+  grid-template-columns: 1fr;
+
+  &__block {
+    height: 300px;
+    background-color: rgb(255, 0, 0);
+    text-align: center;
+    line-height: 300px;
+
+    @each $block, $color in $colors {
+      &--#{$block} {
+        background-color: #{$color};
+      }
+    }
+  }
+
+  @media (min-width: 600px) {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  @media (min-width: 900px) {
+    grid-template-columns: 1fr 1fr 1fr;
```

you could use repeat here
```suggestion
    grid-template-columns: repeat(3, 1fr);
```

---

Okay noted. Thank you have done corrections.